packages/tldraw/src/lib/ui/hooks/useActions.tsx

```diff
@@ -71,6 +72,17 @@ function makeActions(actions: TLUiActionItem[]) {
 	return Object.fromEntries(actions.map((action) => [action.id, action])) as TLUiActionsContextType
 }
 
+function getExportName(editor: Editor, defaultName: string) {
+	const selectedShapes = editor.getSelectedShapes()
+	// When we don't have any shapes selected, we want to use the document name
+	if (selectedShapes.length === 0) {
+		const documentName = editor.getDocumentSettings().name
+		if (documentName === '') return defaultName
+		documentName
```

Comment:
`return documentName` :)

Reply:
Nice catch. I guess my mind is still in Clojure where everything is an expression 😂